applications/ChatGPT/chatgpt/experience_maker/naive.py

```diff
@@ -26,11 +26,9 @@ def make_experience(self, input_ids: torch.Tensor, **generate_kwargs) -> Experie
         value = self.critic(sequences, action_mask, attention_mask)
         r = self.reward_model(sequences, attention_mask)
 
-        reward = compute_reward(r, self.kl_coef, action_log_probs, base_action_log_probs, action_mask=action_mask)
```

Comment:
Did you accidentally remove this?

Reply:
no, My previous thought may be wrong. i will close the PR.